bin/provision-instances

```diff
@@ -0,0 +1,227 @@
+#!/usr/bin/python3
+#
```

Comment:
@bdero All code files should have it - so the other scripts are to be updated too : )